WowPacketParserModule.V2_5_1_38707/Parsers/UpdateHandler.cs

```diff
@@ -159,7 +159,11 @@ public static void HandleUpdateObject(Packet packet)
 
         private static void ReadCreateObjectBlock(Packet packet, CreateObject createObject, WowGuid guid, uint map, object index)
         {
-            ObjectType objType = ObjectTypeConverter.Convert(packet.ReadByteE<ObjectType801>("Object Type", index));
+            ObjectType objType;
+            if (ClientVersion.AddedInVersion(ClientVersionBuild.V2_5_4_42695))
```

Comment:
shouldnt this be RemovedInVersion according to brotalnias comment?

Reply:
I refactored it a bit to be more readable. Clarification - BCC flags PRE 2.5.4, 801 flags POST 2.5.4